walkman.el

```diff
@@ -117,7 +117,7 @@ KEEP-HEADERS is a bool to tell wether or not to keep headers"
         (unless (or walkman-keep-headers keep-headers)
           (delete-region (point-min) (point)))
         (dolist (header headers)
-          (if (and (string= "content-type" (car header))
+          (if (and (compare-strings "content-type" nil nil (car header) nil nil t)
```

Comment:
would it be more readable to do `(string= "content-type" (downcase (car header)))` ?

Reply:
Yeah, I think that's more readable. Updated the MR according to that suggestion :+1: